### PR TITLE
minor fixes

### PR DESCRIPTION
**Pull request by velrest**
- [x] project-from validations
- [x] project-form Bauwekstyp not saved
- [x] building validations
- [x] dwelling validations
- [ ] 1. building entrance is hard set on building creation since there is no form for the 1. entrance yet
- [x]  Coordinates are always sent which results in a 400 if empty
- [x] Linking a new building to a construction project does not work
- [ ] After linking a building via search the scrolling is broken

Depends on #68

### CODE REVIEW

**Comment by velrest** [comment]
The last two issues are a bit big for this pr so they will be addressed separately. @luytena Maybe also something for you for in between tasks.

File: addon/models/building-work.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import Building from "./building";
 import XMLModel from "./xml-model";
 
 export default class BuildingWork extends XMLModel {
-  @tracked kindOfWork;
+  @tracked kindOfWork = BuildingWork.kindOfWorkOptions[0];
```
Comment (luytena) [inline addon/models/building-work.js:7]: Does the select of the validated form now show and use the default value correctly? I never actually looked into that issue last week.

Comment (velrest) [reply addon/models/building-work.js:7]: Hmm good question. I will test this really quick.

**Review by kaldras — APPROVED** [review]

File: addon/models/building-work.js (hunk 1 above)
Comment (velrest) [reply addon/models/building-work.js:7]: Yep works :+1:

Comment (luytena) [reply addon/models/building-work.js:7]: Nice :ok_hand:

**Pull request merged**